src/installer/tests/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.Bundle.Tests/BundleAndRun.cs

```diff
@@ -46,6 +46,21 @@ private void CheckFileNotarizable(string path)
                 .Pass();
         }
 
+        private string MakeUniversalBinary(string path, string rid)
+        {
+            string fatApp = path + ".fat";
+            string arch = BundleHelper.GetTargetArch(rid) == Architecture.Arm64 ? "arm64" : "x86_64";
+
+            Command.Create("lipo", $"-create -arch {arch} {path} -output {fatApp}")
+                .CaptureStdErr()
+                .CaptureStdOut()
+                .Execute()
+                .Should()
+                .Pass();
```

Comment:
I think it's worth adding a comment that this creates a FAT binary with only one MACH-O image in it. So not truly a universal binary. But for the purposes of testing this should be good enough.